shared/featureconfig/config.go

```diff
@@ -123,41 +123,51 @@ func InitWithReset(c *Flags) func() {
 	return resetFunc
 }
 
-// ConfigureBeaconChain sets the global config based
-// on what flags are enabled for the beacon-chain client.
-func ConfigureBeaconChain(ctx *cli.Context) {
-	// Using Medalla as the default configuration for now.
-	params.UseMedallaConfig()
-
-	complainOnDeprecatedFlags(ctx)
-	cfg := &Flags{}
-	if ctx.Bool(devModeFlag.Name) {
-		enableDevModeFlags(ctx)
-	}
+// ConfigureTestnet sets the config according to specified testnet flag
+func ConfigureTestnet(ctx *cli.Context, cfg *Flags) {
```

Comment:
Please do not export this. It shouldn't be used outside featureconfig.

Reply:
ok